src/common/common.js

```diff
@@ -181,7 +181,7 @@ function getDefinition(content) {
   if (def) {
     def = def
     .trim()
-    .replace(/\n|\n\r/g, '')
```

Comment:
Do you know if there was anything specific we were looking for with `\n\r`?

Reply:
Probably crlf but that should have been \r\n anyway.